src/Services/login.service.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+export const loginService = async (userData, requestType) => {
```

Comment:
You can name it authService maybe, because it's being used for both login and signup.

Reply:
Thanks, I will change the name